server/app.js

```diff
@@ -43,6 +45,34 @@ app.post(
   employeeHandlers.createEmployeeHandler
 );
 
+
+
+// --------
+// /clients
+// --------
+
+app.get(
+  buildAPIPath("/clients/:clientID"),
+  clientHandlers.getclientIDPathValidator,
+  checkValidationErrorMiddleware,
+  clientHandlers.getClientByIDHandler
+);
+
+app.patch(
+  buildAPIPath("/clients/:clientID/wallet"),
+  clientHandlers.getAddFundToWalletBodyValidator,
```

Comment:
clientHandlers.getAddFundToWalletBodyValidator
```suggestion
  clientHandlers.addFundToWalletValidatorChain,
```